package.json

```diff
@@ -3,7 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "npm-run-all --parallel dev:*",
+    "dev:next": "next dev -p 3443",
+    "dev:https-link": "local-ssl-proxy --source 443 --target 3443",
```

Comment:
Won't we be forcing everyone to use https in this case? Like if someone isn't working on API related stuff, they still will have to setup the https, is it so? I haven't used it, so I'm not sure. Please clarify.

Newcomers might have to undergo more hurdles setting something up which they might not even what about 😅

Reply:
The https setup is automatic
You can still use the site on localhost:3443, but I would discourage that and encourage devs to start getting on https:// for development too

Reply:
Does that mean localhost:3443 would be on http by default, is it so? Or it'll be https://localhost:3443?

Reply:
http://localhost:3443
https://localhost
https://localhost:443
https://dev.realdevsquad.com

Reply:
Cool